addons/pos_restaurant/static/src/app/screens/floor_screen/floor_screen.js

```diff
@@ -523,7 +522,7 @@ export class FloorScreen extends Component {
                 floor_id: this.activeFloor.id,
             };
         }
-        if (!duplicateFloor && !copyTable) {
+        if (!duplicateFloor) {
```

Comment:
The method _getNewTableNumber seems to not works on duplicate, I know is not from you, but if you can fix it thats cool

![image](https://github.com/user-attachments/assets/30688c75-5508-4808-a6a4-a9e316770e6b)